sshexec fails on windows

````
Caused by: java.lang.ClassCastException: org.apache.maven.wagon.providers.ssh.external.ScpExternalWagon cannot be cast to org.apache.maven.wagon.providers.ssh.jsch.ScpWagon
   at org.codehaus.mojo.wagon.SshExecMojo.execute(SshExecMojo.java:69)
   at org.codehaus.mojo.wagon.AbstractSingleWagonMojo.execute(AbstractSingleWagonMojo.java:64)
   at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:134)
   ... 21 more
````

See the end of #11

--- a/src/main/java/org/codehaus/mojo/wagon/SshExecMojo.java
+++ b/src/main/java/org/codehaus/mojo/wagon/SshExecMojo.java
@@ -22,9 +22,11 @@ package org.codehaus.mojo.wagon;
 import org.apache.maven.plugin.MojoExecutionException;
 import org.apache.maven.plugins.annotations.Mojo;
 import org.apache.maven.plugins.annotations.Parameter;
+import org.apache.maven.wagon.CommandExecutor;
 import org.apache.maven.wagon.Streams;
 import org.apache.maven.wagon.Wagon;
 import org.apache.maven.wagon.WagonException;
+import org.apache.maven.wagon.providers.ssh.jsch.AbstractJschWagon;
 import org.apache.maven.wagon.providers.ssh.jsch.ScpWagon;
 
 /**
@@ -63,7 +65,14 @@ public class SshExecMojo
             {
                 try
                 {
-                    Streams stream = ( (ScpWagon) wagon ).executeCommand( command, true, false );
+                    Streams stream;
+                    if ( wagon instanceof AbstractJschWagon )
+                    {
+                        stream = ( (AbstractJschWagon) wagon ).executeCommand( command, true, false );
+                    } else
+                    {
+                        stream = ( (CommandExecutor) wagon ).executeCommand( command, false );
+                    }
                     this.getLog().info( "sshexec: " + command + " ..." );
                     if ( displayCommandOutputs )
                     {

